etc/collectd.d/nginx.conf

```diff
@@ -5,5 +5,5 @@ LoadPlugin nginx
 <Plugin "nginx">
     # When using non-standard Nginx configurations, replace the below with
     #URL "http://NGINX_HOST:NGINX_PORT/nginx_status"
-    URL "http://localhost/nginx_status"
+    URL "http://local-stackdriver-agent.stackdriver.com:80/nginx_status"
```

Comment:
Is this safe to use? I notice the address resolves correctly on my tests on EC2 and GCE, but not on my local (internal) Google workstation. Maybe that's just due to some paranoid Google setting?